msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/usgov/fairfaxmigrated/TestCase948676.java

```diff
@@ -0,0 +1,127 @@
+
+// Broker authentication with PRT with USGov account with instance_aware=true
+// https://identitydivision.visualstudio.com/Engineering/_workitems/edit/948676
+public class TestCase948676 extends AbstractMsalBrokerTest {
+
+    @Test
+    public void test_948676() throws InterruptedException {
+        final String username = mLoginHint;
+        final String password = LabConfig.getCurrentLabConfig().getLabUserPassword();
+
+        // perform device registration (will obtain PRT in Broker for supplied account)
+        mBroker.performDeviceRegistration(username, password);
+
+        final CountDownLatch latch = new CountDownLatch(1);
+
+        final AcquireTokenParameters parameters = new AcquireTokenParameters.Builder()
+                .startAuthorizationFromActivity(mActivity)
+                .withLoginHint(mLoginHint)
+                .withScopes(Arrays.asList(mScopes))
+                .withCallback(successfulInteractiveCallback(latch))
+                .withPrompt(Prompt.SELECT_ACCOUNT)
+                .build();
+
+        final InteractiveRequest interactiveRequest = new InteractiveRequest(
+                mApplication,
+                parameters,
+                new OnInteractionRequired() {
+                    @Override
+                    public void handleUserInteraction() {
+                        final PromptHandlerParameters promptHandlerParameters = PromptHandlerParameters.builder()
+                                .prompt(PromptParameter.SELECT_ACCOUNT)
+                                .loginHint(username)
+                                .sessionExpected(true)
+                                .consentPageExpected(false)
+                                .speedBumpExpected(false)
+                                .broker(getBroker())
+                                .expectingBrokerAccountChooserActivity(true)
+                                .expectingLoginPageAccountPicker(false)
+                                .build();
+
+                        new AadPromptHandler(promptHandlerParameters)
+                                .handlePrompt(username, password);
+                    }
+                }
+        );
+
+        interactiveRequest.execute();
+        latch.await();
+    }
+
+
+    @Override
+    public LabUserQuery getLabUserQuery() {
+        final LabUserQuery query = new LabUserQuery();
+        query.userType = LabConstants.UserType.CLOUD;
+        query.azureEnvironment = LabConstants.AzureEnvironment.AZURE_US_GOVERNMENT_MIGRATED;
+        return query;
+    }
+
+    @Override
+    public String getTempUserType() {
+        return null;
+    }
+
+    @Override
+    public String[] getScopes() {
+        return new String[]{"User.read"};
+    }
+
+    @Override
+    public String getAuthority() {
+        return mApplication.getConfiguration().getDefaultAuthority().toString();
+    }
+
+    @Override
+    public ITestBroker getBroker() {
+        return new BrokerMicrosoftAuthenticator();
+    }
```

Comment:
You no longer need to override this with my recent changes....let's remove this so that the test can be run with any Broker that would be specified via the build variant.

Reply:
removed the getBroker() function from test case